src/test/java/net/kaczmarzyk/spring/data/jpa/web/WebRequestProcessingContextPathVariableResolverTest.java

```diff
@@ -45,116 +45,143 @@ public class WebRequestProcessingContextPathVariableResolverTest {
 	@Parameters(name = "TestController: {0}")
 	public static Collection testController() {
 		return Arrays.asList(
-				TestControllerWithClassLevelRequestMappingWithValue.class,
-				TestControllerWithClassLevelRequestMappingWithValueAndPathVarWithRegexp.class,
-				TestControllerWithClassLevelRequestMappingWithPath.class,
-				TestControllerWithClassLevelRequestMappingWithPathAndPathVarWithRegexp.class
+				new Object[] { TestControllerWithClassLevelRequestMappingWithValue.class, Boolean.TRUE },
+				new Object[] { TestControllerWithClassLevelRequestMappingWithValueAndPathVarWithRegexp.class, Boolean.TRUE },
+				new Object[] { TestControllerWithClassLevelRequestMappingWithPath.class, Boolean.TRUE },
+				new Object[] { TestControllerWithClassLevelRequestMappingWithPathAndPathVarWithRegexp.class, Boolean.TRUE },
+				new Object[] { TestControllerWithClassLevelRequestMappingWithValue.class, Boolean.FALSE },
+				new Object[] { TestControllerWithClassLevelRequestMappingWithValueAndPathVarWithRegexp.class, Boolean.FALSE },
+				new Object[] { TestControllerWithClassLevelRequestMappingWithPath.class, Boolean.FALSE },
+				new Object[] { TestControllerWithClassLevelRequestMappingWithPathAndPathVarWithRegexp.class, Boolean.FALSE }
 		);
 	}
-	
+
 	private Class<?> testController;
+	private Boolean getAttributes;
 	
-	public WebRequestProcessingContextPathVariableResolverTest(Class<?> testController) {
+	public WebRequestProcessingContextPathVariableResolverTest(Class<?> testController, Boolean getAttributes) {
 		this.testController = testController;
+		this.getAttributes = getAttributes;
 	}
 	
 	@Test
 	public void resolvesPathVariableFromClassLevelRequestMapingAndMethodLevelRequestMapping_empty() {
 		MockWebRequest req = new MockWebRequest("/customers/888");
+		setPathVariableAttributes(req, pathVariables(entry("customerId", "888")));
```

Comment:
similarly as in one of previous comments -- doesn't it change the test logic so that it will fall into the new code only? SHouldn't we add separate tests instead?

CC @jradlica